src/components/Layout/RightSidebar.tsx

```diff
@@ -321,6 +321,34 @@ const RightSidebar = () => {
                 </Tooltip>
               </a>
             ))}
+            <a
+              href={`${location.pathname.replace(/\/$/, '')}/index.md`}
+              className="flex h-5 ui-theme-dark group/markdown-link cursor-pointer"
+              onClick={(e) => {
+                // In development mode, markdown files aren't generated, so show alert immediately
+                if (process.env.NODE_ENV === 'development') {
+                  e.preventDefault();
+                  alert('Markdown files are only available in production builds. Run "yarn build" to generate them.');
+                }
+
+                track('markdown_link_clicked', {
+                  location: location.pathname,
+                });
+              }}
+            >
+              <Tooltip
+                content="View in Markdown"
+                triggerElement={
+                  <img
+                    src="/icons/markdown-mark.svg"
+                    alt="View in Markdown"
+                    className="w-5 h-5 transition-opacity opacity-60 group-hover/markdown-link:opacity-100"
+                  />
+                }
+              >
+                View in Markdown
```

Comment:
"View in Markdown" is inconsistent with the rest of the tooltips here as the surrounding context is "Open in ...", so "Open in View in Markdown" isn't right. Just "Markdown" would be better here

Reply:
I don't agree, because you're not opening in Markdown. I didn't want to fundamentally change the UI, so felt this is in fact better.